delete-rgs.azcli

```diff
@@ -21,3 +21,16 @@ az group list --tag delete --query [].name -o tsv | xargs -otl az group delete -
 # Alternative query filtering for specific tag key/value | This version will NOT ask you to confirm each resource group deletion
 # USE WITH CARE!!!!
 #az group list --query "[?tags.foo=='bar'].name" -o tsv | xargs -otl az group delete -y --no-wait -n
+
+# Mega powers delete across subscriptions and e.g. groups without tags - USE WITH CAUTION
+# Requires the Resource Graph extension (az extension add --name resource-graph)
+# More info: https://docs.microsoft.com/en-us/azure/governance/resource-graph/first-query-azurecli
+# az graph query -q "ResourceContainers | where type == 'microsoft.resources/subscriptions/resourcegroups'| where subscriptionId == '<subscriptionId to target>' \ # Query for RGs in a specific subscription
```

Comment:
Can you define subid and any other params as a var so easy to find what needs to be replaced?

Reply:
I only pulled out subId, as the query itself could be anything and hard to parameterize.